include/seqan3/search/all.hpp

```diff
@@ -68,19 +64,19 @@
  *
  * \endif
  *
- * ### Reference
+ * **Reference:**
  *
  * Kianfar, K., Pockrandt, C., Torkamandi, B., Luo, H., & Reinert, K. (2018).
  *
  * Optimum Search Schemes for Approximate String Matching Using Bidirectional FM-Index. bioRxiv, 301085.
  * https://doi.org/10.1101/301085
  *
- * \if DEV
- * ## K-mer Indices
  *
- * \todo Rewrite landing page.
- * \endif
+ * # K-mer Indices
  *
+ * A k-mer index can be used to efficiently retrieve all occurrences of a certain k-mer in the text.
+ * The k-mer can be either an exact string of length k or it can contain one or more wildcards,
+ * which denote positions of arbitrary characters.
```

Comment:
Do we want to give an example of a k-mer with wildcards? Or refer to the documentation where the use of wildcards is explained?